.github/workflows/docker.yml

```
@@ -0,0 +1,39 @@
+name: Docker Image
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'develop'
+    paths:
+      - '.github/workflows/docker.yml'
+      - 'mk/linux/Dockerfile'
+  schedule:
+    - cron: '0 0 1 * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      -
+        name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+      -
+        name: Build and push
+        uses: docker/build-push-action@v3
```

```suggestion
        uses: docker/build-push-action@v4
```
v4 released Jan 30 https://github.com/docker/build-push-action/releases/tag/v4.0.0